### ISSUE TITLE
fills of chartThemeOverrides not work in cross-filter chart

### ISSUE THREAD

**Issue by YuerLee** [ceolter/ag-grid#5360]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[ ] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[ ] support request => see 'Requesting Community Support'
```

**Current behavior**
Cannot use `fills` in `chartThemeOverrides` to control the chart's series colors in the cross-filter chart. It works in the range chart.

[Cross-filter chart Plnkr](https://plnkr.co/edit/v9NFER3bt2HReGeW?open=main.js&preview)
<img width="1084" alt="image" src="https://user-images.githubusercontent.com/2075566/179181840-557c8655-054e-4520-bcad-bd7a91a0f327.png">

[Range chart Plnkr](https://plnkr.co/edit/dI5h8xwLk8HsaJh8?open=main.js&preview)
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/2075566/179181967-f0d46128-77ed-42c6-8146-cd0bd40ddce8.png">

**Expected behavior**
Can override `fills` value by `chartThemeOverrides` in the cross-filter chart.

**Please tell us about your environment:**
* OS: Windows 10 21H2
* AG Grid version: 28.0.0 (Use latest Plnkr example from the ag-grid website)
* Browser: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36'
* Language: all

**Comment by spyro2000** [comment]
Seems also not working with pivot charts (series.fill/fills are ignored)

**Comment by kiril-matev** [comment]
Hi @spyro2000,

Thanks for reporting this. 

However, this github issue is about cross-filtering charts. Please submit a separate github issue about pivot charts with an example reproducing the issue.

Thank you for your time!

**Comment by YuerLee** [comment]
Fixed in v28.1.0.
